recipes/r-openssl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.9.6' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-openssl
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: openssl_{{ version }}.tar.gz
```

Comment:
FYI this doesn't need to match the filename in the URL. This is just what it is saved as. Typically we have made this the package name and then version. So maybe `r-openssl-{{ version }}.tar.gz`? Really only a suggestion and this only might matter for people building lots of packages offline, but figured I'd mention it.